src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -473,9 +475,19 @@ internal void PopulateAutoCompleteCandidates()
             if (IsDisplayingMLRecommendation)
             {
                 DisplayMachineLearningResults();
+                //Tracking Analytics when raising Node Autocomplete with the Recommended Nodes option selected (Machine Learning)
+                Analytics.TrackEvent(
+                    Actions.Show,
+                    Categories.NodeAutoCompleteOperations,
+                    nameof(NodeAutocompleteSuggestion.MLRecommendation));
             }
             else
             {
+                //Tracking Analytics when raising Node Autocomplete with the Object Types option selected.
+                Analytics.TrackEvent(
+                    Actions.Show,
+                    Categories.NodeAutoCompleteOperations,
+                    nameof(NodeAutocompleteSuggestion.ObjectType));
```

Comment:
My mistake, I thought this is the code flip the mode but this is the actual display code. Updated and I will add tracking where user can flip mode within node autocomplete UI